data/functions/itemized_sync.sql

```diff
@@ -0,0 +1,47 @@
+-- Data syncing scripts
+-- adding a redundant processes to make sure there isn't missing data
+
+-- Receipts
+
+-- looks to see if there are any missing records compared to canonical table
+create or replace function check_schedule_a() returns int as $$
+begin
+    select count(fec_vsum_sched_a.sub_id)
+    from fec_vsum_sched_a
```

Comment:
I know this is a WIP, but remember that it's `fec_vsum_sched_a_vw`, or possibly one of the new views being created as a part of #2324!